src/components/OrganizationSaas/Dashboard/Layout.tsx

```diff
@@ -180,13 +253,57 @@ const OrganizationDashboardLayout: React.FC = () => {
             w='full'
             ml='auto'
             pt='30px'
+            mb='2vh'
           >
             <Outlet />
           </Box>
         </Flex>
       </Wrapper>
+      <PricingModal isOpenModal={isOpenModal} onCloseModal={onCloseModal} />
     </OrganizationProvider>
   )
 }
 
+const PricingModal = ({ isOpenModal, onCloseModal }: { isOpenModal: boolean; onCloseModal: () => void }) => (
+  <Modal isOpen={isOpenModal} onClose={onCloseModal} variant='dashboard-plans'>
+    <ModalOverlay />
+    <ModalContent>
+      <ModalHeader>
+        <Trans>You need to upgrade to use this feature</Trans>
+      </ModalHeader>
+      <ModalCloseButton fontSize='lg' />
+      <ModalBody>
+        {Cards.map((card, idx) => (
+          <PricingCard key={idx} {...card} />
+        ))}
+      </ModalBody>
+
+      <ModalFooter>
+        <Box>
+          <Text>
+            <Trans>If you need more voters, you can select it here:</Trans>
+          </Text>
+          <Select>
+            <option>1-500 members</option>
+          </Select>
+        </Box>
+        <Text>
+          <Trans>
+            Currently tou are subscribed to the 'Your plan' subscription. If you upgrade, we will only charge the yearly
+            difference. In the next billing period, starting on 'dd/mm/yy' you will pay for the new select plan.
+          </Trans>
+        </Text>
+        <Box>
+          <Text>
+            <Trans>Need some help?</Trans>
+          </Text>
+          <Button colorScheme='white'>
+            <Trans>Contact us</Trans>
+          </Button>
```

Comment:
Is the contact us route already implemented? If is, navigate to it

Reply:
It's not implemented yet, but I have set as={ReactRouterLink} so that it's ready. Thanks.